packages/smarthr-ui/src/components/TextLink/stories/TextLink.stories.tsx

```diff
@@ -94,6 +94,18 @@ export const TargetBlank: StoryObj<typeof TextLink> = {
   },
 }
 
+export const Size: StoryObj<typeof TextLink> = {
+  name: 'size',
+  render: (args) =>
+    (['M', 'S'] as const).map((size) => (
+      <p>
+        <TextLink {...args} size={size}>
+          {size}
+        </TextLink>
+      </p>
+    )),
+}
```

Comment:
配列が返っているので型がエラーになっていそうでした。
divとかで囲めばいったんは回避できそうです。

```suggestion
export const Size: StoryObj<typeof TextLink> = {
  name: 'size',
  render: (args) => (
    <div>
      {(['M', 'S'] as const).map((size) => (
        <p key={size}>
          <TextLink {...args} size={size}>
            {size}
          </TextLink>
        </p>
      ))}
    </div>
  ),
}
```